projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.html

```diff
@@ -1,12 +1,18 @@
 <div class="document-filter-container">
-  <div class="form-group document-filter" [formGroup]="documentFilterFormGroup">
-    <input class="form-control document-search"
-      type="search"
-      id="document-search"
-      name="documentSearchFormControl"
-      formControlName="documentSearchFormControl"
-      placeholder="Search by document name">
-  </div>
+  <form class="form-group document-filter" [formGroup]="documentFilterFormGroup">
+    <div class="document-filter__container">
+      <input class="form-control document-search"
+             required
+             type="text"
```

Comment:
@MateiHMCTS - just to check... does setting the type to `text` mean we lose the "magnifying glass" icon that signifies this is a search field?

Reply:
The type="search" was only having the reset functionality built in -- i.e. the cross. This is now mimicked by button type="reset"
The cross isn't working on Firefox, so we had to find a workaround (there's more info about this in the ticket's comments).

Reply:
Magnifying glass is still there

Reply:
Ah - thanks for confirming, @MateiHMCTS... that's great